docs/docs/sourcing-from-kentico-cloud.md

```diff
@@ -1,229 +0,0 @@
----
-title: Sourcing from Kentico Cloud
----
-
-In this guide, you'll go through how to quickly source content for your Gatsby site from [Kentico Cloud](https://kenticocloud.com/).
-
-Kentico Cloud is a hosted CMS that offers you Content as a Service (CaaS) for all your content needs. Using CaaS ensures that your content will be future-proof and reusable, so you can add a mobile app that uses the same content as your Gatsby site without worrying about how it will fit. Kentico Cloud offers CaaS with an easy-to-use editing interface and excellent collaboration features, so all your work on content can happen in one place without requiring technical help for each user.
-
-You can also take advantage of Kentico Cloud's ability to deliver content in multiple languages and create relationships among various content with linked items. However you structure your content in Kentico Cloud, the [source plugin](https://github.com/Kentico/gatsby-source-kentico-cloud) will ensure the proper nodes are created for your Gatsby site.
-
-_Note: For this guide, you'll work from scratch from the [Gatsby default starter](https://github.com/gatsbyjs/gatsby-starter-default) to get a feel for how sourcing from Kentico Cloud works. If you'd like to look at an example of a complete basic site, have a look at the [Kentico Cloud Gatsby starter site](https://github.com/Kentico/gatsby-starter-kentico-cloud), which includes worked examples for querying content._
```

Comment:
We could transfer this section to the pull request because Gatsby starter is already renamed and merged #19203